pkg/operator/csi/credentialsrequestcontroller/credentials_request_controller.go

```diff
@@ -193,3 +192,38 @@ func isProvisioned(cr *unstructured.Unstructured) (bool, error) {
 
 	return provisionedValBool, nil
 }
+
+func shouldSync(cloudCredentialLister operatorv1lister.CloudCredentialLister) (bool, error) {
+	clusterCloudCredential, err := cloudCredentialLister.Get(clusterCloudCredentialName)
+	if err != nil {
+		klog.Errorf("Failed to get cluster cloud credential: %v", err)
+		return false, err
+	}
+
+	isManualMode := func() bool {
+		return clusterCloudCredential.Spec.CredentialsMode == opv1.CloudCredentialsModeManual
+	}
```

Comment:
Why is it a function?

```suggestion
	isManualMode := clusterCloudCredential.Spec.CredentialsMode == opv1.CloudCredentialsModeManual
```